support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -204,6 +225,7 @@ const ContributionThankYou = ({
       email={email}
       createReferralCodes={campaignSettings && campaignSettings.createReferralCodes}
       campaignCode={campaignSettings && campaignSettings.campaignCode}
+      isUsEndOfYearAppeal={isUsEndOfYearAppeal}
```

Comment:
`isUsEndOfYearAppeal` is currently true if the user is in the US (so it does not check if the campaign is live).
This is because we wanted the reminders feature to go live earlier.
I'm just waiting for confirmation that it's ok for the features introduced in this PR to also go live early